dev/tools/proto-to-mapper/Makefile

```diff
@@ -9,6 +9,8 @@ generate-pb: install-protoc-linux
 	git clone https://github.com/googleapis/googleapis.git third_party/googleapis || (cd third_party/googleapis && git reset --hard origin/master && git pull)
 	protoc --include_imports --include_source_info \
 		-I ./third_party/googleapis/ \
+		-I ../../../mockgcp/apis \
+		../../../mockgcp/apis/mockgcp/cloud/networkconnectivity/*/*.proto \
```

Comment:
Can I ask for more context about this, like why we cannot (or prefer not to) put the proto in `./third_party/` (I'm a little bit concerned on the amount of potential duplicated codes we have in cc)

Reply:
So we're adding the generated proto here, because Google doesn't ship one for network connectivity (that I could find).  We happen to be reusing the proto we generated from the OpenAPI here, because it is duplicated.  We could regenerate it here.

The challenge is that I think of mockgcp as separate from this, this is part of a tool that generates KCC code.  mockgcp is a testing tool for GCP.  mockgcp, proto-to-mapper/controllerbuilder and KCC itself could all be in separate repos.